spread.yaml

```diff
@@ -155,6 +155,12 @@ suites:
  tests/spread/general/:
    summary: tests of snapcraft core functionality
 
+ tests/spread/multipass/:
+   summary: tests of snapcraft using multipass
+   # Keep this 18.04 only for now as it is the only stable base.
+   systems: [ubuntu-18.04*]
+   manual: true
```

Comment:
let's try and remove this line to see if the tests actually run.

Reply:
Line removed.